.dockerignore

```diff
@@ -0,0 +1,2 @@
+packages/**/node_modules
+node_modules
```

Comment:
These are the only two ignored paths because we `COPY` the built JS version of the `hub` inside the docker image.